README.md

```diff
@@ -45,6 +45,61 @@ jobs:
           test-results-junit: test-results/results.xml
           code-coverage-cobertura: code-coverage/coverage.xml
 ```
+
+### Run Tests in Parallel
+Run your MATLAB and Simulink tests in parallel (requires Parallel Computing Toolbox&trade;) using the latest release of the required products on a GitHub-hosted runner. To set up the latest release of MATLAB, Simulink, Simulink Test, and Parallel Computing Toolbox on the runner, specify the [Setup MATLAB](https://github.com/matlab-actions/setup-matlab/) action with its `products` input in your workflow. To run the tests in parallel, specify the **Run MATLAB Tests** action with its `use-parallel` input specified as `true`.
+
+```YAML
+name: Run Tests in Parallel
+on: [push]
+jobs:
+  my-job:
+    name: Run MATLAB and Simulink Tests in Parallel
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out repository
+        uses: actions/checkout@v4
+      - name: Set up products
+        uses: matlab-actions/setup-matlab@v2
+        with:
+          products: |
```

Comment:
@mcafaro: We can use either | or > on this line. Is there a preference?

Reply:
Yes, either will work. `>` is probably preferable because it aligns most with our documented API that we accept products "separated by spaces". `|` will include newline characters but we simply ignore them.

`>-` is actually safest because it does not emit a newline character at the end (`>` does). But the newline does not cause any issues for `setup-matlab`.

Reply:
Ok, thanks for this information. I will go ahead with `>`, which also appears on the MATLAB extension page for Azure DevOps.